docker-compose.yml

```
@@ -0,0 +1,10 @@
+version: '3.8'
+
+services:
+  mongo:
+    image: mongo:latest
+    container_name: mongo-dev
+    ports:
+      - "27017:27017" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)